docs/flow/tutorials/scraping-the-agent.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/scraping-the-agent
+title: Scraping the Agent
+weight: 100
+---
+
+Grafana Agent is a telemetry collector and the primary goal is to get telemetry data from one location to another. The goal of this tutorial is to get a basic working example of Grafana Agent working in Flow mode.  
+
+# Prerequisites
+
+* [Docker](https://www.docker.com/products/docker-desktop)
+* Clone the [Agent Repository](https://github.com/grafana/agent) `git clone git@github.com:grafana/agent.git`
```

Comment:
Long term, I wonder if we could find a way to package everything into a single Docker compose file just to remove this one step.

Not a blocker for right now, but it'd be nice if users didn't have to clone the entire repo eventually.

Reply:
I hate the cloning and may wrap it up in a shell script to download the necessary files.

Reply:
I think I will revisit this once we have more of the documentation in place. I think there are far more elegant ways to do it aside from download/clone the repo.